state/validation.go

```diff
@@ -112,6 +105,26 @@ func validateBlock(stateDB dbm.DB, state State, block *types.Block) error {
 		}
 	}
 
+	// Validate block Time
+	if block.Height > 1 {
```

Comment:
should we add a clause Height==1 and require that it's at least the genesis time ? presumably, actually it should be some time *after* the genesis time ?

Reply:
Probably we can enforce only that it's at least the genesis time.